db/migrate/20121206004603_create_states.rb

```diff
@@ -6,5 +6,67 @@ def change
 
       t.timestamps
     end
+
+    State.reset_column_information
+
+    states = {
+      'AL' => 'Alabama',
+      'AK' => 'Alaska',
+      'AZ' => 'Arizona',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.